app.py

```diff
@@ -48,22 +48,44 @@ def __init__(self, name, dir):
     {
         'full_name'         : 'upstream/master',
         'label'             : 'Latest',
-        'allowed_vehicles'  : [copter, plane, rover, sub, tracker, blimp, heli]
+        'allowed_vehicles'  : [copter, plane, rover, sub, tracker, blimp, heli],
+        'artifacts_dir'     : '/latest',
     },
     {
         'full_name'         : 'upstream/Plane-4.3',
         'label'             : 'Plane 4.3 stable',
-        'allowed_vehicles'  : [plane]
+        'allowed_vehicles'  : [plane],
+        'artifacts_dir'     : '/stable',
     },
     {
         'full_name'         : 'upstream/Copter-4.3',
         'label'             : 'Copter 4.3 stable',
-        'allowed_vehicles'  : [copter, heli]
+        'allowed_vehicles'  : [copter, heli],
+        'artifacts_dir'     : '/stable',
     },
     {
         'full_name'         : 'upstream/Rover-4.3',
         'label'             : 'Rover 4.3 stable',
-        'allowed_vehicles'  : [rover]
+        'allowed_vehicles'  : [rover],
+        'artifacts_dir'     : '/stable',
+    },
+    {
+        'full_name'         : 'upstream/Copter-4.4',
+        'label'             : 'Copter 4.4 beta',
+        'allowed_vehicles'  : [copter, heli],
+        'artifacts_dir'     : '/beta',
```

Comment:
Ithink we should be consistent with our directory names and labels/full_name -  either include 4.4 everywhere or remove it everywhere.  Given we may want to run multiple betas in parallel, probably including it here makes sense.